website_mass_mailing_name/tests/test_ui.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from openerp.tests.common import HttpCase
+
+
+class UICase(HttpCase):
+    def test_ui(self):
+        """Test snippet behavior."""
+        self.phantom_js(
+            "/",
+            "openerp.Tour.run('mass_mailing_partner', 'test')",
```

Comment:
`odoo.__DEBUG__.services['web_tour.tour'].run('mass_mailing_partner', 'test')` & `odoo.__DEBUG__.services['web_tour.tour'].tours.mass_mailing_partner.ready` for v10.0?